### ISSUE TITLE
Columns are override/missing when clicking between tables with same name and different columns quantity and files

### ISSUE THREAD

**Issue by roquegarofalo** [HeidiSQL/HeidiSQL#1823]
**Preconditions**
* HeidiSQL version: 12.5.0.6680 (64 bits)
* Database type and version: SQLite
* OS: Windows 10

**Describe the bug**
When open multiples SQLite database files these are shown in the UI explorer. The issue is that columns are override/missing when clicking between tables with same name and different columns quantity and files, the issue is only for "Table:" tab.

**To Reproduce**
Example:
1. Open multiples SQLite files, that contain same tables names but with different columns quantity.
2. On the UI explorer expand the tree of each file.
3. Click on any of table with the same name.
4. Click on "Table:" tab and see how columns are overriden or missing.

> File1.Accounts
>   - Id
>   - FullName
>
> File2.Accounts
>   - Id
>   - FirstName
>   - LastName

_The "Table:" tab in UI shown columns Id and Name of File1._

**Screenshots or Crash reports**
![image](https://github.com/HeidiSQL/HeidiSQL/assets/44882539/909eb431-31e8-4a9d-ba89-c66d4f37a08d)
![image](https://github.com/HeidiSQL/HeidiSQL/assets/44882539/b1ef3ea9-a260-484d-8d0f-590f5611941c)

**Comment by ansgarbecker** [comment]
Is that also happening when you create separate sessions for the both database files?

**Comment by roquegarofalo** [comment]
No, It doesn't happen when open databases files in separate sessions.

![image](https://github.com/HeidiSQL/HeidiSQL/assets/44882539/79deece7-6801-48d9-8281-37d740b8b03d)
